src/Autocomplete/assets/package.json

```diff
@@ -1,10 +1,19 @@
 {
     "name": "@symfony/ux-autocomplete",
-    "description": "JavaScript-powered autocompletion functionality for forms.",
+    "description": "JavaScript Autocomplete functionality for Symfony",
+    "license": "MIT",
+    "version": "2.23.0",
+    "keywords": [
+        "symfony-ux"
+    ],
+    "homepage": "https://ux.symfony.com/autocomplete",
+    "repository": "https://github.com/symfony/ux/tree/2.x/src/Autocomplete",
```

Comment:
Question: should this be the monorepo or the split one ? 

( https://github.com/symfony/ux-autocomplete [...] ) ?

Reply:
I wouldn't mind using monorepo here, as it is a go-to place for proposing contributions

Reply:
Problem is, JsDeliver, Packagist, etc;... do not work well with monorepo... so it depends on "how" this package.json attribute is used (I have no idea so I will trust you guys on this)

Reply:
Fair enough then 👍